mozilla-mobile/focus-android/l10n.toml

```diff
@@ -0,0 +1,17 @@
+basepath = "."
+
+locales = [
+]
```

Comment:
I haven't populated this list yet. I assume we can add all the locales we currently have here?

Reply:
That would be my assumption as well, since it's how the other projects in android-l10n work.
Side-question: not sure how we are going to handle release locales vs shipping locales. Like in Fenix?

Reply:
Currently we still use this locale list in the repository and Focus/Klar release builds still use this to filter down the list of locales:
https://github.com/mozilla-mobile/focus-android/blob/master/tools/l10n/locales.py

We could continue using this. Or I am also happy to change it to whatever makes sense for you.

Regarding this list here, I'll add a commit and populate this list with all locales we already have, okay?

Reply:
yes, that sounds good, thanks!